tools/diff-processor/breaking_changes/mock_schema_diff_test.go

```diff
@@ -0,0 +1,36 @@
+package breaking_changes
+
+// MockSchemaDiff implements the diff.SchemaDiff interface for testing
+type MockSchemaDiff struct {
```

Comment:
I would prefer to not mock per go/choose-test-double?polyglot=go#prefer-real-implementations, but I'm not blocking on it.

Reply:
ack, this is checked upstream in another integration test. I would like to keep these simple and relevant to the logic they need to check.